CONTRIBUTING.md

```diff
@@ -35,14 +35,14 @@ Consult the Table of Contents below, and jump to the desired section.
 
 If you are a Kong Enterprise customer, contact the Enterprise Support channels
 by opening an Enterprise support ticket on
-[https://support.konghq.com](https://support.konghq.com/).
+[support](https://support.konghq.com/).
 
 If you are experiencing a P1 issue, please call the [24/7 Enterprise Support
 phone line](https://support.konghq.com/hc/en-us/articles/115004921808-Telephone-Support)
 for immediate assistance, as published in the Customer Success Reference Guide.
 
 If you are interested in becoming a Kong Enterprise customer, please visit
-https://konghq.com/kong-enterprise-edition/ or contact us at
+[https://konghq.com/kong-enterprise-edition/](https://konghq.com/kong-enterprise-edition/) or contact us at
```

Comment:
This change is unnecessary, GitHub's markdown parser will already produce a link with the previous version. Also, this makes the line go beyond the 80 columns limit enforced in this document.